example/assets/yarn.lock

```diff
@@ -1,2761 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Any particular reason to switch from yarn to npm lock files in the example?

Reply:
Nope. I assumed it was connected to the old brunch workflow and removed it along with `example/assets/brunch-config.js`

Reply:
Its not related to brunch. Does mix phx.new create an npm lockfile though? I guess we can switch to it if it does.

Reply:
yes `mix phx.new myapp` makes a file `myapp/assets/package-lock.json`